airflow/www/templates/airflow/ti_log.html

```diff
@@ -51,7 +63,15 @@ <h4>{{ title }}</h4>
     // Animation speed for auto tailing log display.
     const ANIMATION_SPEED = 1000;
     // Total number of tabs to show.
-    const TOTAL_ATTEMPTS = "{{ logs|length }}";
+    const TOTAL_ATTEMPTS = getMetaValue('total-attempts');
+    // Exectuin date
```

Comment:
I think most of these variable names are self-explanatory, and we don't need the comments.

Reply:
I apply your suggestion.